spec/collator.html

```diff
@@ -135,9 +135,9 @@ <h1>Internal Slots</h1>
       </p>
 
       <ul>
-        <li>The first element of [[SortLocaleData]][[&lt;_locale_&gt;]].[[co]] and [[SearchLocaleData]][[&lt;_locale_&gt;]].[[co]] must be *null*.</li>
+        <li>The first element of [[SortLocaleData]].[[&lt;_locale_&gt;]].[[co]] and [[SearchLocaleData]].[[&lt;_locale_&gt;]].[[co]] must be *null*.</li>
```

Comment:
this was really confusing to read at first (either form) because it's implying that `[[SortLocaleData]]` is a first-class thing. Perhaps it would be clearer if it said something like this:
```suggestion
        <li>The first element in any object `_obj_` of _obj_.[[SortLocaleData]].[[&lt;_locale_&gt;]].[[co]] and [[SearchLocaleData]].[[&lt;_locale_&gt;]].[[co]] must be *null*.</li>
```
?

Reply:
That would need to be %Collator%.[[SortLocaleData]], not _obj_.[[SortLocaleData]]. (Especially because that uses _obj_ in the definition of what _obj_ refers to)

Reply:
sure, if there's a specific object here then that's the obvious choice :-) it's just very unclear to me to have an internal slot used like this.